assets/js/jquery-2.2.4.min.js

```diff
@@ -0,0 +1,4 @@
+/*! jQuery v2.2.4 | (c) jQuery Foundation | jquery.org/license */
```

Comment:
@Agakit please remove `jquery-2.2.4.min.js` - we already have `jquery.min.js` in `assets/js` directory